docs/source/configuration.rst

```diff
@@ -472,6 +472,12 @@ You can set the dbt project directory, profiles directory and profile with:
     profiles_dir = <relative or absolute path to the directory that contains the profiles.yml file>
     profile = <dbt profile>
 
+.. note::
+
+    If the `profiles_dir` setting is omitted, SQLFluff will look for the profile
+    in the default location, which varies by operating system, e.g. Windows, Mac,
+    Linux.
```

Comment:
Is it worth being more specific about where this default location is?

Or is it a standard location for dbt configs (If so may be worth providing a link to the relevant dbt doc section)?

Reply:
I had the same thought. It is documented [here](https://docs.getdbt.com/dbt-cli/configure-your-profile#connecting-to-your-warehouse-using-the-command-line), but it only mentions the Unix-style default location (`~/.dbt/`). There's no mention of Windows, which is the odd 🦆 here. About Windows, this is the only info I can find: https://stackoverflow.com/questions/63127291/unable-to-get-dbt-folder-or-profiles-yml-file-after-installing-python-3-8-an

Because of the above, I'm thinking it's better not to link to incomplete docs, but I'm okay either way. What do you think?

Reply:
Let me have a quick dig through the dbt code to see if I can track it down 🕵️

Reply:
uses `os.path.expanduser("~")`
![image](https://user-images.githubusercontent.com/80432516/145225849-ffc1445b-39b4-4d03-b44a-0636eba0b9fb.png)

Which for windows is not very helpful
![image](https://user-images.githubusercontent.com/80432516/145226027-9cfa7097-495e-40cd-a9ba-c600425df6cb.png)

Reply:
However, dbt does mention that you can find the default location via `dbt debug --config-dir`
(https://docs.getdbt.com/dbt-cli/configure-your-profile#:~:text=Advanced%3A%20Customizing%20a%20profile%20directory%23)

So let's just add a note along the lines of:

(you can determine your default profile directory via `dbt debug --config-dir`)

Reply:
Will do! I'm AFK, will add this later today.